### ISSUE TITLE
Using reserved words as arguments should give a clear warning

### ISSUE THREAD

**Issue by ArneBachmann** [evhub/coconut#386]
But stops with a parsing error without clear hint on the reason

**Comment by evhub** [comment]
@ArneBachmann What sort of reserved words are you talking about? Can you give an example error message?

**Comment by ArneBachmann** [comment]
don't remember but could've been "from"

**Comment by evhub** [comment]
Fixed on `coconut-develop>=1.5.0-post_dev35`:
```python
Coconut Interpreter:
(enter 'exit()' or press Ctrl-D to end)
>>> return = 1

CoconutParseError: parsing failed (possible cause: invalid use of the keyword "return") (line 1)
  return = 1
>>>
```

Also:
```python
>>> if a == b
     pass

CoconutParseError: parsing failed (possible cause: misplaced newline (maybe missing ':')) (line 1)
  if a == b
          ^
>>>
```
And:
```python
>>> if a = b:
     pass

CoconutParseError: parsing failed (possible cause: misplaced assignment (maybe should be '==')) (line 1)
  if a = b:
       ^
>>>
```
